### ISSUE TITLE
Monstesr are still crashing - not driving well - too easily spill over

### ISSUE THREAD

**Issue by patrickdenglerhobbisoft** [patrickdenglerhobbisoft/WorldDefense#22]

**Comment by helmut-g** [comment]
Cars can tip over when they drive automatically or when the user takes control. 
To improve the cars I would do this:
1. Define the set of car models that should be in the first version of the game.
2. Tweak the car wheel positions and vehicle physics properties. The wheel positions and sizes still look odd. Some cars have a high wheel suspension length which moves the center of mass up and makes it easier to roll over.
3. Add some acceleration/deceleration code to reduce speed in curves (when driving automatically)

Regarding 3: There is already an AdjustAcceleration code in VehicleObject. I will have a look and try to improve it.

**Comment by helmut-g** [comment]
I added some code for breaking before curves:
The MonsterContentObject stores the current target position and the next target position (to allow the vehicle steering to anticipate curves). There is also commented out code to create a zig-zag test route to debug tight curves.
The VehicleObject checks if the car is near a curve. If it is, it steps onto the breaks to decelerate to a target speed depending on the curve tightness.

I have tested it with orange jeep. The curve behavior (speed, brake force, etc.) is controlled by constants in the VehicleObject. We could adjust this per car or we simply tune it for the most unstable car.
